package.json

```diff
@@ -38,5 +38,8 @@
     "grunt-template-jasmine-istanbul": "^0.4.0",
     "grunt-template-jasmine-requirejs": "^0.2.3",
     "grunt-markdox": "^1.2.1"
+  },
```

Comment:
The build should pass even without this.
Proof: https://travis-ci.org/TalAter/annyang/builds/132315158